src/main/webapp/app/exercises/programming/manage/update/wizard-mode/programming-exercise-update-wizard.component.html

```diff
@@ -3,19 +3,19 @@
         class="form-step"
         *ngIf="currentStep >= 1"
         [isImport]="isImport"
-        [titleNamePattern]="titleNamePattern"
+        [titleNamePattern]="infoStepInputs.titleNamePattern"
         [programmingExercise]="programmingExercise"
-        [shortNamePattern]="shortNamePattern"
-        [invalidRepositoryNamePattern]="invalidRepositoryNamePattern"
-        [invalidDirectoryNamePattern]="invalidDirectoryNamePattern"
-        [updateRepositoryName]="updateRepositoryName"
-        [updateCheckoutDirectory]="updateCheckoutDirectory"
-        [refreshAuxiliaryRepositoryChecks]="refreshAuxiliaryRepositoryChecks"
-        [auxiliaryRepositoryDuplicateNames]="auxiliaryRepositoryDuplicateNames"
-        [auxiliaryRepositoryDuplicateDirectories]="auxiliaryRepositoryDuplicateDirectories"
-        [exerciseCategories]="exerciseCategories"
-        [existingCategories]="existingCategories"
-        [updateCategories]="updateCategories"
+        [shortNamePattern]="infoStepInputs.shortNamePattern"
```

Comment:
couldn't you just pass `infoStepsInputs` instead of the single attributes?

Reply:
Done